example_systems/CA_AZ/combined_settings.yml

```diff
@@ -1061,10 +1061,10 @@ cost_multiplier_technology_map:
   Advanced nuclear: [Nuclear_*, Nuclear_mid, Nuclear_low, Nuclear_high]
   Biomass: [Biopower_CofireOld, Biopower_CofireNew, Biopower_Dedicated]
   MSW - landfill gas: []
-  Wind: [LandbasedWind]
-  Wind offshore: [OffShoreWind]
+  Wind: [LandbasedWind_Class4]
```

Comment:
Can you also apply these changes to the `resources.yml` file in the "settings" folder?